docs_template/source/guides_explanations/environments.md

```diff
@@ -4,101 +4,67 @@ of your favourite data analysis library. The same holds for debugging errors tha
 only because your coauthor uses a slightly different setup.
 
 The solution is to have isolated environments on a per-project basis.
-[Conda environments](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
-allow you to do precisely this. This page describes them a little bit and explains their
-use.
+[Pixi environments](https://pixi.sh/latest/tutorials/python/) allow you to do precisely
+this. This page describes them a little bit and explains their use.
 
 The following commands can either be executed in a terminal or the Powershell (Windows).
 
 ### Using the environment
 
-In the installation process of the template a new environment was created if it was not
-explicitly declined. It took its specification from the environment.yml file in your
-projects root folder.
-
-To activate it, execute:
-
-```console
-$ conda activate <env_name>
-```
-
-Repeat this step every time you want to run your project from a new terminal window.
+Pixi environments are created automatically when a command is run via `pixi run [...]`,
```

Comment:
Maybe like:

- These templates ship with a pre-configured environment. 
- You can inspect the contents in the `[tool.pixi.xxx]` sections
- When you type `pixi run ...` or `pixi install`, the packages are downloaded ...

Completely delete the next section `### Setting up...`. 

Then continue the `Updating ...` section:

Precise versions of packages are pinned down in the file `pixi.lock`, ensuring reproducibility. If you want to update ...